### PR TITLE
small documentation additions

### PR DESCRIPTION
**Pull request by raphael-proust**

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -5,18 +5,20 @@ PKGS=\
...
 	-I ${BLD}/ppx/.ppx_lwt.objs/byte \
 	-I ${BLD}/react/.lwt_react.objs/byte \
+	-I ${BLD}/domain/.lwt_domain.objs/byte \
 	-I ${BLD}/unix/.lwt_unix.objs/byte
 
 MLIS=\
 	$(wildcard ${SRC}/core/*.mli) \
 	$(wildcard ${SRC}/ppx/*.mli) \
-	$(wildcard ${SRC}/react/*.mli) \
+	$(wildcard ${SRC}/domain/*.mli) \
```
Comment (smorimoto) [inline docs/Makefile:21]: Is it intentional that react is removed here?

**Review by smorimoto — APPROVED** [review]

**Comment by smorimoto** [comment]
I'd like to go with this, apart from the one thing I commented on!

File: docs/Makefile (hunk 1 above)
Comment (raphael-proust) [reply docs/Makefile:22]: That's an error. I wanted to add an entry rather than to replace one. It's fixed now.

Comment (smorimoto) [reply docs/Makefile:22]: Looks good now!

**Pull request merged**